index.html

```diff
@@ -12,6 +12,9 @@
     {% include description.html %}
     
     {% include projects.html %}
+
+    {% include education.html %}
```

Comment:
Do we need the education section? This wasn't discussed in the issue. It seems like a feature from an old MLH portfolio website which is redundant now because we are not making an individual portfolio website but instead we are making a portfolio for the whole pod. I think it might be good to remove it.